tsconfig.settings.json

```diff
@@ -12,7 +12,9 @@
         "declaration": true,
```

Comment:
The rename is a personal preference.  I think it also gets you looser JSON syntax checking (e.g. so trailing commas are allowed).

Reply:
Got it. We named it this way to make it easier to select tsconfig files (`*tsconfig.json`)

Reply:
Whatever works for you - I use `tsconfig*.json` :wink:.